hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketDeleteRequest.java

```diff
@@ -135,12 +151,16 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           DeleteBucketResponse.newBuilder().build());
 
       // Add to double buffer.
-      omClientResponse = new OMBucketDeleteResponse(volumeName, bucketName,
-          omResponse.build());
+      omClientResponse = new OMBucketDeleteResponse(omResponse.build(),
+          volumeName, bucketName);
+      LOG.debug("Deleted bucket:{} in volume:{}", bucketName, volumeName);
     } catch (IOException ex) {
+      omMetrics.incNumBucketDeleteFails();
+      LOG.error("Delete bucket failed for bucket:{} in volume:{}", bucketName,
```

Comment:
Can we move back logging to before?

Why that is done like that way is, logging to be done outside the lock.

Reply:
done